src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+  margin: 0;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url('../images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+
+    &:nth-child(-n + 4) {
+      background-image: url('../images/star-active.svg');
+      background-repeat: no-repeat;
+      background-position: center;
```

Comment:
```suggestion
```

Reply:
In general to make this styles flexible you have to add modifier class to be able change active-stars count by changing only html class